doc/sphinx/how-to/how-to-enable-graphics-core22-on-a-device.md

```diff
@@ -0,0 +1,455 @@
+```
+Node: /dev/dri/card1
+├───Driver: amdgpu (AMD GPU) version 3.54.0 (20150101)
+│   ├─── ...
+├───Device: PCI 1002:747e Advanced Micro Devices, Inc. [AMD/ATI] Device 747e
+│   └───Available nodes: primary, render
+├───Framebuffer size
+│   ├───Width: [0, 16384]
+│   └───Height: [0, 16384]
+├───Connectors
+|   |   ...
+│   ├───Connector 3
+│   ├───Object ID: 101
+│   ├───Type: HDMI-A
+│   ├───Status: connected
+│   ├───Physical size: 530x300 mm
+│   ├───Subpixel: unknown
+│   ├───Encoders: {3}
+│   ├───Modes
+│   │   ├───1920x1080@60.00 preferred driver phsync pvsync 
+│   │   ├───1920x1080@60.00 driver phsync pvsync 16:9 
+│   │   ├───...
+|   |   ...
+├───Encoders
+│   ├───Encoder 0
+│   │   ├───Object ID: 82
+│   │   ├───Type: TMDS
+│   │   ├───CRTCS: {0, 1, 2, 3}
+│   │   └───Clones: {0}
+|   |   ...
+├───CRTCs
+│   ├───CRTC 0
+│   │   ├───Object ID: 72
+│   │   ├───Legacy info
+│   │   │   ├───Mode: 1920x1080@60.00 preferred driver phsync pvsync 
+│   │   │   └───Gamma size: 256
+|   |   ...
+└───Planes
+    ├───Plane 0
+    │   ├───Object ID: 40
+    │   ├───CRTCs: {3}
+    │   ├───Legacy info
+    │   │   ├───FB ID: 0
+    |   ...
+```
+You want to make sure that you have:
+- 1 DRM Node (?) TOOD
+- 1 or more Connector(s)
+- 1 or mor CRTC(s)
+- 1 or more Plane(s)
+
+Finally, If you have a display connected, then one of your `Connector`s should read `Status: connected`. This `Connector` will also list many `Modes` that
+the  display can use.
+
+#### Troubleshooting
+If `drm_info` does *not* produce results similar to those above, then this
+indicates that your kernel **is NOT yet ready** to support the GPU. This
+means that kernel enablement is required before ubuntu frame can be enabled.
+
+### 3. Driver Support Check
+Finally, we will need to check that you have the proper userspace drivers
+installed on the system, namely:
+- `EGL` (`libEGL.so.1`)
+- `GLES` (`libGLESv2.so.2`) 
+- `GBM` (`libgbm.so.1`)
+
+While *Mir* can be enabled on platforms that do not support `GBM/KMS`, this
+requires a paltform implementation to be written in *Mir*. This work is out
+of the scope of this guide.
+
+## How to assemble a `graphics-core22` provider snap
+With the prerequisites out of the way, it is time to assemble a
+`graphics-core22` snap.
+
+The snap itself must provide a handful libraries
+which fall into the following four categories:
+
+- Graphics interfaces
+  - `libEGL.so.1`
+  - `libva.so.2`
+  - `libvulkan.so.1`
+  - `libGLESv2.so.2`
+
+- GBM/Kms interfaces
+  - `libdrm.so.2`
+  - `libgbm.so.1`
+
+- Wayland support
+  - `libwayland-client.so.0`
+  - `libwayland-server.so.0`
+
+- X support (optional)
+  - `libX11-xcb.so.1`
+  - `libXau.so.6`
+  - `libxcb-dri2.so.0`
+  - `libxcb-dri3.so.0`
+  - `libxcb-present.so.0`
+  - `libxcb.so.1`
+  - `libxcb-sync.so.1`
+  - `libxcb-xfixes.so.0`
+  - `libXdmcp.so.6`
+  - `libxshmfence.so.1`
+
+`libEGL`, `libGLESv2` and `libgbm` should be provided by the vendors drivers.
```

Comment:
```suggestion
`libEGL`, `libGLESv2` and `libgbm` should be provided by the vendor's drivers.
```

Reply:
Also, `libvulkan` *may* be provided by the vendor. I'm not entirely sure how best to word that.

Reply:
We could mentioned that it _may_ be provided. I think that at least prompts the user to check in some way.